foreman-proxy/foreman-proxy.service

```diff
@@ -8,6 +8,7 @@ Type=forking
 User=foreman-proxy
 ExecStart=/usr/share/foreman-proxy/bin/smart-proxy
 PIDFile=/run/foreman-proxy/foreman-proxy.pid
+TimeoutSec=infinity
```

Comment:
It looks like 0 was changed to infinity in version 229 (the latest), so I guess 0 might be safest for now: http://article.gmane.org/gmane.comp.sysutils.systemd.devel/35665